### DISCUSSION TITLE
FS22 - Italia Emilia Ext

### DISCUSSION THREAD

**Discussion by Ulanawius** [KillBait/FS19_AutoDrive_MapImages#22] [General]
I would like to have the Italia Emilia Ext map added to the Editor.

**Comment by KillBait** [comment]
No problem, i'll get onto is as soon as i can

**Comment by KillBait** [comment]
Upload Images just now for the SP/MP versions, apologies for the delay

**Reply by Ulanawius** [reply]
Thank you very much! Great work on the editor :-) I like it a lot better than the FS19 version :-)
